python/doc/examples/reliability_sensitivity/reliability/plot_nais.py

```diff
@@ -215,17 +202,18 @@ def fourBranch(x):
     dr.setColor(col[i])
     graph.add(dr)
 
-# sphinx_gallery_thumbnail_number = 2
-view = View(graph)
+# %%
+_ = View(graph)
 
 # %%
 # Draw the frontiers only
 # -----------------------
 #
-# The following graph enables to understand the progression of the algorithm from the mean value of the initial distribution to the limit state function:
+# The following graph enables to understand the progresison of the algorithm:
```

Comment:
> > progresison

> fixed

This is not fixed in the current state of the PR. Maybe you lost the commit in the merge?
```suggestion
# The following graph enables to understand the progression of the algorithm:
```